lab2.js

```diff
@@ -86,33 +104,71 @@ var hello = {
 // speak, and method (that you'll place on the prototype) called
 // sayHello.
 
-function SentientBeing () {
-  // TODO: specify a home planet and a language
+
+// TODO: specify a home planet and a language
   // you'll need to add parameters to this constructor
+
+function SentientBeing (homePlanet, language) {
+  this.homePlanet = homePlanet;
+  this.language = language;
 }
 
 // sb is a SentientBeing object
-function sayHello (sb) {
+SentientBeing.prototype.sayHello = function sayHello (sb) {
     // TODO: say hello prints out (console.log's) hello in the
     // language of the speaker, but returns it in the language
     // of the listener (the sb parameter above).
     // use the 'hello' object at the beginning of this exercise
     // to do the translating
+   if (sb.language == 'romulan') {
```

Comment:
this works. you can also do 
console.log(hello[this.language]);
return hello[sb.language];
and then you don't need these ifs.